.eslintrc.json

```diff
@@ -34,7 +35,16 @@
 				"no-constant-condition": 0,
 				"no-nested-ternary": 0,
 				"jsx-a11y/click-events-have-key-events": 0,
-				"jsx-a11y/no-static-element-interactions": 0
+				"jsx-a11y/no-static-element-interactions": 0,
+				"comment-length/limit-tagged-template-literal-comments": [
```

Comment:
This rule just pertains to tagged template literals, but eslint-plugin-comment-length offers [multiple rules for different kinds of comments](https://www.npmjs.com/package/eslint-plugin-comment-length#rules). Do you intend this to limit the length of all comment types?

Reply:
Thanks! I added all three rules.